src/components/forms/ProxyDeviceForm.tsx

```diff
@@ -250,8 +250,8 @@ const ProxyDeviceForm: FC<Props> = ({ formik, project }) => {
         device.bind,
         "Whether to bind the listen address to the instance or host",
         (value) => {
-          if (value === "instance" && device.nat === "true") {
-            void formik.setFieldValue(`devices.${index}.nat`, "false");
+          if (value === "instance" && device.nat !== undefined) {
```

Comment:
I think `device.nat !== undefined` check is not necessary. We also wouldn't have to worry about unnecessary re-renders since React checks if state value has changed before re-rendering (for primitive values).

Reply:
Good point, removing the extra check.